awx/ui_next/src/screens/Template/Survey/SurveyQuestionEdit.jsx

```diff
@@ -8,12 +13,23 @@ export default function SurveyQuestionEdit({ survey, updateSurvey }) {
   const [formError, setFormError] = useState(null);
   const history = useHistory();
   const match = useRouteMatch();
+  const { search } = useLocation();
+  const queryParams = new URLSearchParams(search);
+  const questionVariable = queryParams.get('question_variable');
 
   if (!survey) {
```

Comment:
I know this is not part of this PR, but to me, this should be handled in TemplateSurvey.jsx where we fetch the survey.